### ISSUE TITLE
Does the renderer  support texture image with transparency ?

### ISSUE THREAD

**Issue by tjusxh** [facebookresearch/pytorch3d#737]
Given a texture image with transparency. I want to render the 3D model with a transparent texture image. How to do it? Thanks.

**Comment by nikhilaravi** [comment]
Do you mean you have an RGBA texture image? Currently we only support RGB texture. Do you only need the forward pass for rendering?

**Comment by tjusxh** [comment]
Yes, I have an RGBA texture image. I only need the forward pass for rendering.

**Comment by tjusxh** [comment]
> Do you mean you have an RGBA texture image? Currently we only support RGB texture. Do you only need the forward pass for rendering?

Thank you for quickly replying.
Yes, I have an RGBA texture image. I only need the forward pass for rendering.

**Comment by nikhilaravi** [comment]
You could try loading the texture as a `TexturesUV` class - w e support `(H, W, C)` maps as explained here: https://github.com/facebookresearch/pytorch3d/blob/4046677cf1f8a6e5e9f9e8ef43e5c25200183c64/pytorch3d/renderer/mesh/textures.py#L605-L607

Then you could writing your own simple shader which doesn't compute an alpha mask separately but uses the RGBA value directly. 


```
class RGBAShader(nn.Module):
    def __init__(
        self,
        device: Device = "cpu",
    ) -> None:
        super().__init__()

    def forward(self, fragments: Fragments, meshes: Meshes, **kwargs) -> torch.Tensor:
        texels = meshes.sample_textures(fragments)
        images = get_rgba(colors, fragments)
        return images
```

Where the `get_rgba` could be a simple modification of the `hard_rgb_blend` function: https://github.com/facebookresearch/pytorch3d/blob/4046677cf1f8a6e5e9f9e8ef43e5c25200183c64/pytorch3d/renderer/blending.py#L26-L70
